index.yaml

```
@@ -1,6 +1,75 @@
 apiVersion: v1
 entries:
   cluster-autoscaler:
+  - apiVersion: v2
+    appVersion: 1.21.0
+    created: "2021-07-08T14:14:58.153453+01:00"
+    description: Scales Kubernetes worker nodes within autoscaling groups.
+    digest: 4a6418c2a33d3b380a1396efa885ec7f8fe1ade4a8c597fefcce419cbc3d98dc
+    home: https://github.com/kubernetes/autoscaler
+    icon: https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png
+    maintainers:
+    - email: e.bailey@sportradar.com
+      name: yurrriq
+    - email: mgoodness@gmail.com
+      name: mgoodness
+    - email: guyjtempleton@googlemail.com
+      name: gjtempleton
+    - email: scott.crooks@gmail.com
+      name: sc250024
+    name: cluster-autoscaler
+    sources:
+    - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
+    type: application
+    urls:
+    - https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-9.10.2/cluster-autoscaler-9.10.2.tgz
+    version: 9.10.2
+  - apiVersion: v2
+    appVersion: 1.21.0
+    created: "2021-07-08T09:39:32.195117+01:00"
+    description: Scales Kubernetes worker nodes within autoscaling groups.
+    digest: 81c51e509c6b2947c436e57046b93a83b49307e565a4827c81d109ac76a56bf1
+    home: https://github.com/kubernetes/autoscaler
+    icon: https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png
+    maintainers:
+    - email: e.bailey@sportradar.com
+      name: yurrriq
+    - email: mgoodness@gmail.com
+      name: mgoodness
+    - email: guyjtempleton@googlemail.com
+      name: gjtempleton
+    - email: scott.crooks@gmail.com
+      name: sc250024
+    name: cluster-autoscaler
+    sources:
+    - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
+    type: application
+    urls:
+    - https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-9.10.01/cluster-autoscaler-9.10.01.tgz
+    version: 9.10.01
```

I'd suggest manually fixing the version here before merging.

```suggestion
    version: 9.10.1
```

---

Good suggestion, done as suggested.